colpali_engine/models/late_interaction/colidefics/processing_colidefics.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import annotations
+
+from typing import List, Optional
+
+from PIL import Image
+from transformers import BatchEncoding, Idefics2Processor
+
+from colpali_engine.utils.processing_utils import BaseVisualRetrieverProcessor
+
+
+class ColIdefics2Processor(BaseVisualRetrieverProcessor, Idefics2Processor):
+    def __init__(self, image_processor):
+        BaseVisualRetrieverProcessor.__init__(self)
+        Idefics2Processor.__init__(self, image_processor)
+
+    def process_images_idefics(
```

Comment:
shoud just be called process_images then process_queries to fit with the abstract class